posthog/clickhouse/client/execute_async.py

```diff
@@ -240,43 +235,25 @@ def enqueue_process_query_task(
     query_status = QueryStatus(id=query_id, team_id=team.id, start_time=datetime.datetime.now(datetime.timezone.utc))
     manager.store_query_status(query_status)
 
-    # Skip cache if refresh requested
-    if not refresh_requested:
```

Comment:
Did I miss this logic being moved to somewhere else?

Reply:
This is now via query runner: If you tell it "cache or async" you'll get the cache already if present. Thus we can remove this here. Anything triggering an async calculation should also get fresh results (unless it runs with same query_id and into the async query cache).

Reply:
Yup, I see now, had to scroll down in the diff